src/XRP/reliable-submission-xrp-client.ts

```diff
@@ -57,7 +57,7 @@ class ReliableSubmissionXRPClient implements XRPClientDecorator {
       transactionHash,
     )
     const { lastLedgerSequence } = rawTransactionStatus
-    if (lastLedgerSequence === 0) {
+    if (!lastLedgerSequence) {
```

Comment:
This is `number | undefined`, so I switched the check so it also type guards against `undefined`.